### PR TITLE
feat: match subdomain for ssr-opengraph

### PR DESCRIPTION
**Pull request by preschian**
changes:
- match subdomain for ssr-opengraph-beta
- add flush cache key to flush cache for image-beta

### CODE REVIEW

**Comment by vikiival** [comment]
I leave it for Leo

Screenshot 2023-04-04 at 11 37 06

File: ssr-opengraph-beta/README.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+```
+npm install
+npm run dev
+```
+
+```
+npm run deploy
+```
```
Comment (roiLeo) [inline ssr-opengraph-beta/README.md:8]: nice readme! keep it up 😉

File: image-beta/src/index.ts [hunk 2]
```diff
@@ -18,8 +18,9 @@ app.all('/ipfs/*', async (c) => {
   const path = url.pathname.replace('/ipfs/', '');
 
   const request = c.req;
+  const flushCache = '2023-04-04-a'; // change the value to flush the cache
```
Comment (roiLeo) [inline image-beta/src/index.ts:21]: can't this be done in a cron way? to avoid changing it often

File: image-beta/src/utils/cloudflare-images.ts [hunk 3]
```diff
@@ -43,6 +43,8 @@ export async function uploadToCloudflareImages({
   );
   const uploadStatus = uploadCfImage.status;
 
+  console.log('uploadStatus', uploadStatus);
```
Comment (roiLeo) [inline image-beta/src/utils/cloudflare-images.ts:46]: is it needed to debug?

**Review by roiLeo — COMMENTED** [review]
> I leave it for Leo
> 
> Screenshot 2023-04-04 at 11 37 06

😨 oh no

I've asign myself since I am not familiar with this stack.

Isn't the same thing as #55? but for beta? why do we need to have different project?

**Comment by roiLeo** [comment]
> correct, previously I had an incident about that [kodadot/nft-gallery#4891](https://github.com/kodadot/nft-gallery/issues/4891). basically similar to `image` and `image-beta`. we test on `-beta` first, and once stable we update the original

Aren't we going to have a problem we new domain name? 
should we duplicate this code to make it work for `canary.*` domain? (main kodadot)

File: image-beta/src/index.ts (hunk 2 above)
Comment (roiLeo) [reply image-beta/src/index.ts:21]: Aight, weird flex but ok

**Review by roiLeo — APPROVED** [review]

**Comment by yangwao** [comment]
> should we duplicate this code to make it work for `canary.*` domain? (main kodadot)

I mean having extra canary would be nice if things will be breaking really quickly on beta env. Right now it's ok to have two env

**Pull request merged**